aderyn_core/src/detect/detector.rs

```diff
@@ -442,6 +447,15 @@ pub enum IssueSeverity {
     High,
 }
 
+/// When different contexts report different number of instances for the same file, this is a merge
+/// conflict and we need a resolution strategy.
+pub enum MergeConflictResolutionStrategy {
```

Comment:
@TilakMaddy could you please remove this as discussed, and use only intersection?

Reply:
Until then, I'l draft it ... Let me know if you disagree

Reply:
Actually I'll do what you said - remove this strategy enum.

When you come back later, we'll see